types/luxon/src/datetime.d.ts

```diff
@@ -32,6 +32,8 @@ export interface ToRelativeOptions extends ToRelativeCalendarOptions {
      * @default 0
      */
     padding?: number;
+    /** If omitted, the method will pick the unit. */
```

Comment:
The method also picks the best unit when this is given too. How about:
```suggestion
    /** A subset of units that the method will pick the best one from */
```

Reply:
Should the comment better reflect all 3 behaviors?

```
A single unit or an array of units. If an array is supplied, the method will pick the best one to use from the array. If omitted, the method will pick the unit from a default set.
```

Optionally is it of any advantage to try and reuse wording from the actual package?
https://github.com/moment/luxon/pull/926/files#diff-5146f40a8c8111b14e5ab3407eff80c7a6d2f3c92a4bf4ccd4647bff32d47e38R1850

Reply:
Yeah that's good with me too!